docker/test-cpu-local-falkordb-only-entrypoint.sh

```
@@ -0,0 +1,13 @@
+
+set -ex
+
+MAYBE_RAPIDS="echo 'no rapids'"
+if [[ "$RAPIDS" == "1" ]]; then
+    source activate rapids
+    MAYBE_RAPIDS="source activate rapids"
+else
+    source /opt/pygraphistry/pygraphistry/bin/activate
+fi
+
+./bin/test.sh $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)